vowpalwabbit/parse_args.cc

```diff
@@ -280,14 +278,14 @@ void parse_diagnostics(options_i& options, VW::workspace& all)
   bool help = false;
   bool skip_driver = false;
   std::string progress_arg;
+  size_t upper_limit = 0;
   option_group_definition diagnostic_group("Diagnostic");
   diagnostic_group.add(make_option("version", version_arg).help("Version information"))
       .add(make_option("audit", all.audit).short_name("a").help("Print weights of features"))
       .add(make_option("progress", progress_arg)
                .short_name("P")
                .help("Progress update frequency. int: additive, float: multiplicative"))
-      .add(make_option("quiet", all.logger.quiet).help("Don't output diagnostics and progress updates"))
-      .add(make_option("limit_output", all.logger.upper_limit).help("Avoid chatty output. Limit total printed lines"))
+      .add(make_option("limit_output", upper_limit).help("Avoid chatty output. Limit total printed lines"))
```

Comment:
not sure if this one belongs here or with the others below (where you add the new one)

Reply:
Good catch, it definitely should go there